### ISSUE TITLE
False positive when url contains query string

### ISSUE THREAD

**Issue by bramkragten** [43081j/eslint-plugin-lit#66]
When you have a template like:
```ts
html`<a href="/?query=${query}">Link</a>`
```

It will give the following errors:
```
Template contained invalid HTML syntax, error was: missing-whitespace-between-attributes eslint(lit/no-invalid-html)

Template contained invalid HTML syntax, error was: unexpected-character-in-attribute-name eslint(lit/no-invalid-html)
```

**Comment by 43081j** [comment]
You should probably be doing:

```ts
html`<a href=${`/?query=${query}`}`>Link</a>`;
```

so lit understands you're interpolating a value. im not sure how lit deals with trying to interpolate part of a value like that, maybe it understands it fine.. 

as for the error though regardless, its the html parser we use giving that. ill take a look either way when i get chance

**Comment by danielvanmil** [comment]
It looks like lit-html/element allows this and so it seems correct. Example in the playground: [](https://lit.dev/playground/#project=W3sibmFtZSI6Im15LWVsZW1lbnQudHMiLCJjb250ZW50IjoiaW1wb3J0IHtMaXRFbGVtZW50LCBodG1sLCBjc3N9IGZyb20gJ2xpdCc7XG5pbXBvcnQge2N1c3RvbUVsZW1lbnQsIHByb3BlcnR5fSBmcm9tICdsaXQvZGVjb3JhdG9ycy5qcyc7XG5cbkBjdXN0b21FbGVtZW50KFwibXktZWxlbWVudFwiKVxuZXhwb3J0IGNsYXNzIE15RWxlbWVudCBleHRlbmRzIExpdEVsZW1lbnQge1xuICAvLyBTdHlsZXMgYXJlIHNjb3BlZCB0byB0aGlzIGVsZW1lbnQ6IHRoZXkgd29uJ3QgY29uZmxpY3Qgd2l0aCBzdHlsZXNcbiAgLy8gb24gdGhlIG1haW4gcGFnZSBvciBpbiBvdGhlciBjb21wb25lbnRzLiBTdHlsaW5nIEFQSSBjYW4gYmUgZXhwb3NlZFxuICAvLyB2aWEgQ1NTIGN1c3RvbSBwcm9wZXJ0aWVzLlxuICBzdGF0aWMgc3R5bGVzID0gY3NzYFxuICAgIDpob3N0IHtcbiAgICAgIGRpc3BsYXk6IGlubGluZS1ibG9jaztcbiAgICAgIHBhZGRpbmc6IDEwcHg7XG4gICAgICBiYWNrZ3JvdW5kOiBsaWdodGdyYXk7XG4gICAgfVxuICAgIC5wbGFuZXQge1xuICAgICAgY29sb3I6IHZhcigtLXBsYW5ldC1jb2xvciwgYmx1ZSk7XG4gICAgfVxuICBgO1xuXG5cbiAgLy8gRGVmaW5lIHJlYWN0aXZlIHByb3BlcnRpZXMtLXVwZGF0aW5nIGEgcmVhY3RpdmUgcHJvcGVydHkgY2F1c2VzXG4gIC8vIHRoZSBjb21wb25lbnQgdG8gdXBkYXRlLlxuICBAcHJvcGVydHkoKSBncmVldGluZyA9IFwiSGVsbG9cIjtcbiAgQHByb3BlcnR5KCkgcGxhbmV0ID0gXCJXb3JsZFwiO1xuICBAcHJvcGVydHkoKSBsZXR0ZXJUID0gXCJ0XCI7XG5cbiAgLy8gVGhlIHJlbmRlcigpIG1ldGhvZCBpcyBjYWxsZWQgYW55IHRpbWUgcmVhY3RpdmUgcHJvcGVydGllcyBjaGFuZ2UuXG4gIC8vIFJldHVybiBIVE1MIGluIGEgc3RyaW5nIHRlbXBsYXRlIGxpdGVyYWwgdGFnZ2VkIHdpdGggdGhlIGBodG1sYFxuICAvLyB0YWcgZnVuY3Rpb24gdG8gZGVzY3JpYmUgdGhlIGNvbXBvbmVudCdzIGludGVybmFsIERPTS5cbiAgLy8gRXhwcmVzc2lvbnMgY2FuIHNldCBhdHRyaWJ1dGUgdmFsdWVzLCBwcm9wZXJ0eSB2YWx1ZXMsIGV2ZW50IGhhbmRsZXJzLFxuICAvLyBhbmQgY2hpbGQgbm9kZXMvdGV4dC5cbiAgcmVuZGVyKCkge1xuICAgIHJldHVybiBodG1sYFxuICAgICAgPHNwYW4gQGNsaWNrPSR7dGhpcy50b2dnbGVQbGFuZXR9XG4gICAgICAgID4ke3RoaXMuZ3JlZXRpbmd9XG4gICAgICAgIDxzcGFuIGNsYXNzPVwicGxhbmUke3RoaXMubGV0dGVyVH1cIj4ke3RoaXMucGxhbmV0fTwvc3Bhbj5cbiAgICAgIDwvc3Bhbj5cbiAgICBgO1xuICB9XG5cbiAgLy8gRXZlbnQgaGFuZGxlcnMgY2FuIHVwZGF0ZSB0aGUgc3RhdGUgb2YgQHByb3BlcnRpZXMgb24gdGhlIGVsZW1lbnRcbiAgLy8gaW5zdGFuY2UsIGNhdXNpbmcgaXQgdG8gcmUtcmVuZGVyXG4gIHRvZ2dsZVBsYW5ldCgpIHtcbiAgICB0aGlzLnBsYW5ldCA9IHRoaXMucGxhbmV0ID09PSBcIldvcmxkXCIgPyBcIk1hcnNcIiA6IFwiV29ybGRcIjtcbiAgfVxufVxuIn0seyJuYW1lIjoiaW5kZXguaHRtbCIsImNvbnRlbnQiOiI8IURPQ1RZUEUgaHRtbD5cbjxoZWFkPlxuICA8c2NyaXB0IHR5cGU9XCJtb2R1bGVcIiBzcmM9XCIuL215LWVsZW1lbnQuanNcIj48L3NjcmlwdD5cbiAgPHN0eWxlPlxuICAgIC5tYXJzIHtcbiAgICAgIC0tcGxhbmV0LWNvbG9yOiByZWQ7XG4gICAgfVxuICA8L3N0eWxlPlxuPC9oZWFkPlxuPGJvZHk-XG4gIDxteS1lbGVtZW50PjwvbXktZWxlbWVudD5cbiAgPGhyIC8-XG4gIDxteS1lbGVtZW50IGNsYXNzPVwibWFyc1wiIHBsYW5ldD1cIk1hcnNcIj48L215LWVsZW1lbnQ-XG48L2JvZHk-XG4ifSx7Im5hbWUiOiJwYWNrYWdlLmpzb24iLCJjb250ZW50Ijoie1xuICBcImRlcGVuZGVuY2llc1wiOiB7XG4gICAgXCJsaXRcIjogXCJeMi4wLjBcIixcbiAgICBcIkBsaXQvcmVhY3RpdmUtZWxlbWVudFwiOiBcIl4xLjAuMFwiLFxuICAgIFwibGl0LWVsZW1lbnRcIjogXCJeMy4wLjBcIixcbiAgICBcImxpdC1odG1sXCI6IFwiXjIuMC4wXCJcbiAgfVxufSIsImhpZGRlbiI6dHJ1ZX1d)

**Comment by 43081j** [comment]
yup sorry i took so long to get back to this, ill try have a look into this weekend.

you're right, it should work fine
